Cypress/cypress/integration/Parallel_Tests/Create_Workflow/uploadWorkflow.spec.js

```diff
@@ -26,40 +26,12 @@ describe("Testing the upload Workflow with correct workflow manifest and target
     cy.chooseWorkflow(3, "", `sample-workflow-${workflowNamespace}.yaml`);
     cy.wait(500);
     cy.get("[data-cy=ControlButtons] Button").eq(1).click();
-    cy.wait(1000); // Waiting for Workflow Details to get filled
+    cy.wait(1000);
```

Comment:
We can have this comment back for future reference

Reply:
I don't think we are editing or filling in any workflow details anymore. I can undo it if required?